src/site/layouts/health_care_local_facility.drupal.liquid

```diff
@@ -127,13 +127,16 @@
               <div data-widget-type="facility-detail" data-facility="{{ fieldFacilityLocatorApiId | widgetFacilityDetail | escape }}"></div>
             </div>
 
-            {% if fieldMedia.entity.image != empty %}
-              <!-- facility image -->
               <div class="usa-width-one-half">
-                {% assign image = fieldMedia.entity.image %}
-                <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="{{ image.derivative.width }}" height="{{ image.derivative.height }}">
+                <!-- facility image -->
+                {% if fieldMedia.entity.image != empty %}
+                  {% assign image = fieldMedia.entity.image %}
+                  <a target="_blank" id="google-map-link-image">
```

Comment:
This doesn't have an `href`. Is one added somewhere else?

Reply:
Yes. It's added in `./src/applications/static-pages/facilities/FacilityMapWidget.jsx`

Reply:
Ok. We probably want to add `rel="noopener noreferrer"`, too. Normally we don't need to do that in templates, but since there's no href at build time, it won't get added.

It's also possible that this will trip some accessibility tests and we might be required to add a dummy value.